front/src/routes/scene/edit-scene/Settings.jsx

```diff
@@ -0,0 +1,93 @@
+import cx from 'classnames';
+import { Localizer, Text } from 'preact-i18n';
+import { useState } from 'preact/compat';
+import { TagsInput } from 'react-tag-input-component';
+
+const Settings = ({ ...props }) => {
+  const [cardOpened, setCardOpened] = useState(false);
+
+  const openCloseCard = () => {
+    setCardOpened(!cardOpened);
+  };
+
+  return (
+    <div class="col">
+      <div class="card">
+        <div class="card-status bg-green" />
+        <div class="card-header" onClick={openCloseCard}>
+          <i
+            class={cx('fe', 'mr-2', {
+              'fe-chevron-right': !cardOpened,
+              'fe-chevron-down': cardOpened
+            })}
+          />
+          <h4 class="text-center card-title ">
+            <Text id="editScene.settings" />
+          </h4>
+          <div class="card-options" />
+        </div>
+        <div
+          class="card-body"
+          style={{
```

Comment:
Use classes instead of direct style. I'm surprised ESLint is not saying something?

Reply:
Fixed